### ISSUE TITLE
Docu fix: `result = trainer.run()`

### ISSUE THREAD

**Issue by carlocagnetta** [aai-institute/tianshou#1008]
### Issue Description

In the provided code examples, it appears that the training function is not being used properly. Specifically, in both cases cited by the code-line links, the variable named `result` is being assigned a trainer class which is never used.

### Proposed Solution

To address this issue, I suggest changing the class instance from `result` to `trainer` and assign the ouput of its function `run()` to the variable `result`; using it as follows:

```python
# Before
result = OnpolicyTrainer()
print(result)

# After
trainer = OnpolicyTrainer()
result = trainer.run()
print(result)

```

or more efficiently:

```python
result = OnpolicyTrainer().run()
print(result)
```

### Affected Code Lines

https://github.com/thu-ml/tianshou/blob/ea48cc298905ad37fa67bd05b76f55079e1905c2/docs/02_notebooks/L6_Trainer.ipynb#L193-L204

https://github.com/thu-ml/tianshou/blob/ea48cc298905ad37fa67bd05b76f55079e1905c2/docs/02_notebooks/L7_Experiment.ipynb#L241-L252

**Comment by carlocagnetta** [comment]
@MischaPanch @opcode81 This is the issue we discussed

**Comment by carlocagnetta** [comment]
resolved by #1009
